pkg/constants/constants.go

```diff
@@ -79,6 +79,9 @@ const (
 	// this annotation will be cleared
 	WorkspaceStopReasonAnnotation = "controller.devfile.io/stopped-by"
 
+	// WebhookRestartedAtAnnotation holds the the time (unixnano) of when the webhook pod was restarted
```

Comment:
We can't tell for sure when pod is restarted, it's deployment work.
```suggestion
	// WebhookRestartedAtAnnotation holds the time (unixnano) of when the webhook server was forced to restart by controller
```

Reply:
fixed!